tinyagentos/worker/agent.py

```
@@ -114,6 +114,10 @@ def __init__(
         self._state_dir = state_dir or default_state_dir()
         self._signing_key: bytes | None = load_signing_key(self._state_dir)
 
+        # Worker update-check service (background version polling).
+        # Created eagerly so callers can inspect state; started in run().
+        self._update_service: "WorkerUpdateService | None" = None
```

Ruff (0.15.21)

[error] Undefined name `WorkerUpdateService`

(F821)